imports/ui/components/setup/SetUpAcadDetail.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Select from 'react-select';
+
+import CourseDropdown from './dropdown/CourseDropdown.jsx';
+import AcadCohortDropdown from './dropdown/AcadCohortDropdown.jsx';
+import PrevEduDropdown from './dropdown/PrevEduDropdown.jsx';
+
+import { createNewStudent } from '../../../api/database-controller/student/methods';
+
+export default class SetUpAcadDetail extends React.Component {
```

Comment:
Good segmentation of functions for clarity @IamVacer !! @walrys  and I will work on styling it, the flow has been tested and works like a charm! Will tell you about bugs if we find any!